### ISSUE TITLE
WebUI: When changing between the Home and Jobs tabs I get logged out.

### ISSUE THREAD

**Issue by marianne013** [ic-hep/pdm#388]
Not good.

**Comment by marianne013** [comment]
Correction: I don't get logged out, but it goes back to the default "Please select a site screen" and I have to select the sites again.

**Comment by alexanderrichards** [comment]
Yes this is expected behaviour but your not the first person to find this annoying so I agree we should remember the previous locations.

**Comment by marianne013** [comment]
Right now the only way to see if a copy job has succeeded is to change tabs.
